pkg/pgp/pgp.go

```diff
@@ -72,6 +73,18 @@ func ExtractPublicKey(entity *openpgp.Entity) ([]byte, error) {
 
 }
 
+func GetFingerprintFromPublicKey(content []byte) (string, error) {
+
```

Comment:
nit: remove this extra line

Reply:
The inclusion of a space at the beginning/end of method matches the rest of the application. We can remove the spaces across the board in a future enhancement